base/CMakeLists.txt

```diff
@@ -152,16 +152,47 @@ find_file(JACKSON2_JAXB_ANNOTATIONS_JAR
 )
 
 execute_process(
-    COMMAND xmvn-resolve jakarta.xml.bind:jakarta.xml.bind-api:4
-    OUTPUT_VARIABLE JAXB_API_JAR
+    COMMAND awk -F= "$1==\"ID\" { print $2 ;}" /etc/os-release
+    OUTPUT_VARIABLE DISTRO
     OUTPUT_STRIP_TRAILING_WHITESPACE
 )
```

Comment:
Generally I'd avoid heavy scripting in CMake since it's not a proper scripting language so sometimes you might need to invoke a shell explicitly and escape some characters to execute some commands, which could make it harder to maintain. But in this particular case it seems to be simple enough so it's fine, although there's a [concern](https://unix.stackexchange.com/questions/432816/grab-id-of-os-from-etc-os-release) about using `awk` for this purpose.